golden/size-of-pactvalue-guard/golden

```diff
@@ -0,0 +1 @@
+[374,{"moduleName":{"namespace":"_P^MBfTBZT@65Rr>Ppuxv!~/-P#f氵*c","name":"+*^j<kTgEsi$Q/x4$艙ThB%"},"name":"q%>4jhdd_!@BZu"}]
```

Comment:
not much coverage here ... and seems to be covered elsewhere (ie list golden has `{"moduleName":{"namespace":"𐂈Oa%/4q4qekz","name":"W@=@Z3~^4nbOLvcﳟHr~3vrH>&MM"},"name":"kcK"}`), so do we need this?

Reply:
Added golden tests for each type of literal and guard.